### PR TITLE
added scripts to build off both rhel7 and centos7

### PR DESCRIPTION
**Pull request by tparikh**
As we setup CI/CD pipeline for Quay, we may need to switch the source image for quay container images. We should achieve this code change for Jenkins job in the app-interface repository and not the upstream quay repository.

https://issues.jboss.org/browse/PROJQUAY-24

Signed-off-by: Tejas Parikh <tparikh@redhat.com>

### CODE REVIEW

**Comment by tparikh** [comment]
[test]

File: scripts/app_sre_build_deploy.sh [hunk 1]
```diff
@@ -1,25 +1,5 @@
...
-
-# push the image
-skopeo copy --dest-creds "${QUAY_USER}:${QUAY_TOKEN}" \
-    "docker-daemon:${IMG}" \
-    "docker://${QUAY_IMAGE}:latest"
-
-skopeo copy --dest-creds "${QUAY_USER}:${QUAY_TOKEN}" \
-    "docker-daemon:${IMG}" \
-    "docker://${QUAY_IMAGE}:${GIT_HASH}"
+# DO NOTHING
```
Comment (thomasmckay) [inline scripts/app_sre_build_deploy.sh]: Is this file needed for something?

File: scripts/app_sre_pr_check.sh [hunk 2]
```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 set -exv
 
-BASE_IMG="quay"
-
-IMG="${BASE_IMG}:latest"
-
-BUILD_CMD="docker build" IMG="$IMG" make app-sre-docker-build
+# DO NOTHING
```
Comment (thomasmckay) [inline scripts/app_sre_pr_check.sh]: File needed?

File: scripts/app_sre_build_deploy.sh (hunk 1 above)
Comment (tparikh) [reply scripts/app_sre_build_deploy.sh]: The file is needed for now because the app-sre pipeline looks for it. Once the PR is merged, I will be able to update the Jenkins job and remove this file.

File: scripts/app_sre_pr_check.sh (hunk 2 above)
Comment (tparikh) [reply scripts/app_sre_pr_check.sh]: The file is needed for now because the app-sre pipeline looks for it. Once the PR is merged, I will be able to update the Jenkins job and remove this file.
